docs/network-flow-visibility.md

```diff
@@ -352,6 +358,22 @@ commands:
 ./infra/vagrant/push_antrea.sh --flow-collector <externalFlowCollectorAddress>
 ```
 
+## IPFIX Collector
+
+The go-ipfix collector can be built from [go-ipfix library](https://github.com/vmware/go-ipfix). It is used to collect, decode and log the IPFIX records. To deploy a released version of the go-ipfix collector, please choose one deployment manifest from the list of releases. For any given release <TAG> (e.g. v0.1.0), you can deploy the collector as follows:
```

Comment:
`please choose one deployment manifest from the list of releases.` -> `please choose one deployment manifest from the list of releases (supported after v0.5.1).`